src/Command/UpdateSignalementDesordresCommand.php

```diff
@@ -0,0 +1,129 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:update-signalement-desordres',
+    description: 'Recompute composition desordres for signalement',
+)]
+class UpdateSignalementDesordresCommand extends Command
+{
+    public function __construct(
+        private SignalementManager $signalementManager,
+        private DesordrePrecisionRepository $desordrePrecisionRepository,
+        private DesordreCompositionLogementLoader $desordreCompositionLogementLoader,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $signalements = $this->addSignalementsByDesordrePrecisionSlug(
+            'desordres_type_composition_logement_piece_unique_superficie'
+        );
+        $signalements = array_merge($signalements, $this->addSignalementsByDesordrePrecisionSlug(
+            'desordres_type_composition_logement_plusieurs_pieces_aucune_piece_9'
+        ));
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_piece_unique_hauteur')
+        );
+        $signalements = array_merge($signalements, $this->addSignalementsByDesordrePrecisionSlug(
+            'desordres_type_composition_logement_plusieurs_pieces_hauteur'
+        ));
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_cuisine_collective_oui')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_cuisine_collective_non')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_douche_collective_oui')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_douche_collective_non')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_wc_collectif_oui')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_wc_collectif_non')
+        );
+        $signalements = array_merge(
+            $signalements,
+            $this->addSignalementsByDesordrePrecisionSlug('desordres_type_composition_logement_wc_cuisine_ensemble')
+        );
```

Comment:
on peut pas factoriser en itérant sur un tableaux de désordres précisions ?

Reply:
yes, c'est fait